apps/api-runtime/src/main/java/com/akto/parsers/HttpCallParser.java

```diff
@@ -267,6 +267,10 @@ public List<HttpResponseParams> filterHttpResponseParams(List<HttpResponseParams
             }
 
             boolean cond = HttpResponseParams.validHttpResponseCode(httpResponseParam.getStatusCode());
+            if (httpResponseParam.getSource().equals(HttpResponseParams.Source.POSTMAN)) {
```

Comment:
We will allow non 2xx responses to be ingested using postman?

Reply:
good catch. Fixing this.